lib/ae_page_objects/elements/collection.rb

```diff
@@ -66,18 +71,21 @@ def item_class_at(index)
 
     def item_xpath
       @item_xpath ||= begin
-        query_args = eval_locator(@item_locator)
-
-        #
-        # Use the { exact: true } setting for XPath selectors that use "XPath.is".  For example, given the XPath
-        #   XPath::HTML.descendant(:div)[XPath.text.is('Example Text')]
-        # the resulting path will be
-        #   .//div[./text() = 'Example Text']
-        # instead of
-        #   .//div[contains(./text(), 'Example Text')]
-        # See https://github.com/jnicklas/capybara#exactness for more information.
-        #
-        query_args += [{:exact => true}] if query_args.first.to_sym == :xpath
+        query_args = eval_locator(@item_locator).dup
+
+        if query_args.first.to_sym == :xpath
+          #
+          # Use the { exact: true } setting for XPath selectors that use "XPath.is".  For example, given the XPath
+          #   XPath::HTML.descendant(:div)[XPath.text.is('Example Text')]
+          # the resulting path will be
+          #   .//div[./text() = 'Example Text']
+          # instead of
+          #   .//div[contains(./text(), 'Example Text')]
+          # See https://github.com/jnicklas/capybara#exactness for more information.
+          #
+          query_args.push({}) unless query_args.last.is_a?(::Hash)
+          query_args.last[:exact] = true unless query_args.last.key?(:exact)
```

Comment:
(Irrelevant to this PR) I don't think we should be afraid to use 1.9+ only stuff here. This version no longer supports 1.8, and so if something needs to be backported, it can be fixed when porting. Otherwise functionality requiring 1.9+ could never be introduced if that is the logic.

Futhermore, ruby 1.8 has been EOL since July 21, 2014 (https://www.ruby-lang.org/en/news/2014/07/01/eol-for-1-8-7-and-1-9-2/) and 1.9 EOL since February 23, 2015 (https://www.ruby-lang.org/en/news/2014/01/10/ruby-1-9-3-will-end-on-2015/). As a result, neither of these versions are something we should continue to put any effort into supporting unless we had the need to for our own legacy code -- I don't believe any such code exists.